vendor/modules.txt

```diff
@@ -267,6 +267,7 @@ k8s.io/apiextensions-apiserver/pkg/features
 # k8s.io/apimachinery v0.0.0 => k8s.io/apimachinery v0.0.0-20190612205821-1799e75a0719
 k8s.io/apimachinery/pkg/labels
 k8s.io/apimachinery/pkg/types
+k8s.io/apimachinery/pkg/util/strategicpatch
```

Comment:
We're not actually adding/removing anything from the vendor here.  Why did this file change?  Do we want to include these changes?

Reply:
I think it changed because we started to use the referenced package as a direct dependency now, whereas it was consumed transitively only before.